src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: $border-radius;
```

Comment:
The variable `$border-radius` is used here, but ensure that it is defined somewhere in your SCSS files. If it's not defined, this will cause a compilation error.